src/include/storage/data_table.h

```diff
@@ -264,12 +266,21 @@ class DataTable : public AbstractTable {
   // INTEGRITY CHECKS
   //===--------------------------------------------------------------------===//
 
-  bool CheckNulls(const storage::Tuple *tuple) const;
+  bool CheckNotNulls(const storage::Tuple *tuple, oid_t column_idx) const;
+  bool MultiCheckNotNulls(const storage::Tuple *tuple,
+                          std::vector<oid_t> cols) const;
+
```

Comment:
Is it possible to delete CheckNotNulls? Then when you're going to call single check null, you could add the "{}" wrap to convert it to a vector and just use MultiCheckNotNulls